borg/archiver.py

```diff
@@ -1399,7 +1440,7 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='ARCHIVE',
                                type=location_validator(archive=True),
                                help='name of archive to create (must be also a valid directory name)')
-        subparser.add_argument('paths', metavar='PATH', nargs='+', type=str,
+        subparser.add_argument('paths', metavar='PATH', nargs='*', type=str,
```

Comment:
ok, good to know. not a showstopper I'ld say, it can be just placed differently to get it working. 

Usually I'ld place options to the left, sometimes (when editing a commandline) it is also convenient if they can be appended, but putting them in the middle is a bit unusual anyway.